modules/nw-ovn-kubernetes-ansible-migration-about.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-ansible-migration-about_{context}"]
+= Using the Ansible playbook to migrate to the OVN-Kubernetes network plugin
+
+As a cluster administrator, you can use an Ansible playbook to migrate from the OpenShift SDN Container Network Interface (CNI) network plugin to the the OVN-Kubernetes plugin for your cluster.
+
+The Ansible playbook includes the following set of playbooks that each represent a stage in the migration process:
+
+. Prechecks
+. Migrate
+. Reboot
+. Post-migration
+
+[IMPORTANT]
+====
+If you run a playbook as a non-cluster administrator, the playbook fails.
+====
+
+If a playbook fails, check the playbook output for the source of the error. Resolve the error and then rerun the Ansible playbook.
+
+.Prerequisites
+
+* You set a bastion server.
+* You installed `ansible`, `jq`, `pip3`, and `python3` packages.
+** To install the `ansible` package, see "Getting started with playbook" in the Additional resources section.
+* You set the `KUBECONFIG` environment variable.
+* Optional: See "Migrating to the OVN-Kubernetes network plugin by using the offline migration method" for a list of other prerequisites. Although the Prechecks playbook checks these prerequsities, you can meet them in advance of the playbook check.
+* You logged in to the {hybrid-console} and opened the link:https://console.redhat.com/ansible/ansible-dashboard[Ansible Automation Platfor] web console.
+* From the *Automation Hub > Token management* page of the Ansible Automation Platform web console, you created an API token and configured Red{nbsp}Hat Automation Hub as the primary source for content. See link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/1.2/html/getting_started_with_red_hat_ansible_automation_hub/proc-create-api-token[Creating the Red{nbsp}Hat Automation Hub API token].
+
+.Procedure
+
+. Download the `openshift` Ansible collection from the link:https://console.redhat.com/ansible/automation-hub[Ansible automation hub] web console page.
+
+. Enter the following command in the `ansible-galaxy` command-line interface to install the downloaded `openshift` Ansible collection file:
++
+[source,terminal]
+----
+$ ansible-galaxy collection install openshift
+----
+
+. Verify that `openshift` Ansible collection was successfully installed by entering the following command:
++
+[source,terminal]
+----
+$ ansible-galaxy collection list openshift
+----
++
+.Example output
++
+[source,terminal]
+----
+# /home/myuser/.ansible/collections/ansible_collections
+Collection               Version
+------------------------ -------
+openshift                4.0.1 
```

Comment:
Do I need to document the FQCN section in https://developers.redhat.com/learning/learn:ansible:getting-started-ansible-content-collections/resource/resources:finding-and-installing-collections-and-using-them-playbooks?source=sso but fit the content's use for the openshift collection?